src/UI/StructureInspector.cpp

```diff
@@ -91,28 +91,28 @@ void StructureInspector::btnCloseClicked()
 
 void StructureInspector::drawPopulationRequirements()
 {
-	Renderer& r = Utility<Renderer>::get();
-	float posX = rect().x() + 10.0f;
-	float posY = rect().y() + 85.0f;
-
-	r.drawText(*FONT_BOLD, "Population Required", rect().x() + 10, posY, 255, 255, 255);
-
-	posY += 20;
-
-	if (mStructure->populationRequirements()[0] > 0)
-	{
-		std::string format = string_format("Workers: %i/%i", mStructure->populationAvailable()[0], mStructure->populationRequirements()[0]);
-		Color color = mStructure->populationAvailable()[0] >= mStructure->populationRequirements()[0] ? Color::White : Color::Red;
-		r.drawText(*FONT, format, posX, posY, color.red(), color.green(), color.blue());
-		posY += 10;
+	Renderer& renderer = Utility<Renderer>::get();
+
+	auto position = rect().startPoint() + NAS2D::Vector{10, 85};
+	renderer.drawText(*FONT_BOLD, "Population Required", position, NAS2D::Color::White);
+
+	const std::array<std::string, 2> populationTypes = {{
```

Comment:
What happens if you declare this as:
```cpp
const std::array<std::string, 2> populationTypes{
"Workers",
"Scientists"
};
```
?

Reply:
Seems to work just fine.